clients/admin-ui/cypress/e2e/datasets.cy.ts

```diff
@@ -1,13 +1,146 @@
 describe("Dataset", () => {
-  describe("List of datasets view", () => {
-    beforeEach(() => {
-      cy.intercept("GET", "/api/v1/dataset", { fixture: "datasets.json" });
+  beforeEach(() => {
+    cy.intercept("GET", "/api/v1/dataset", { fixture: "datasets.json" });
+    cy.intercept("GET", "/api/v1/dataset/*", { fixture: "dataset.json" });
+    cy.intercept("GET", "/api/v1/data_category", {
+      fixture: "data_category.json",
     });
+  });
+
+  describe("List of datasets view", () => {
     it("Can navigate to the datasets list view", () => {
       cy.visit("/");
       cy.getByTestId("nav-link-Datasets").click();
       cy.getByTestId("dataset-table");
       cy.getByTestId("dataset-row-demo_users_dataset_4");
+      cy.url().should("contain", "/dataset");
+    });
+
+    it("Can navigate to the datasets view via URL", () => {
+      cy.visit("/dataset");
+      cy.getByTestId("dataset-table");
+    });
+
+    it("Can load an individual dataset", () => {
+      cy.visit("/dataset");
+      cy.getByTestId("load-dataset-btn").should("be.disabled");
+      cy.getByTestId("dataset-row-demo_users_dataset").click();
+      cy.getByTestId("load-dataset-btn").should("not.be.disabled");
+      cy.getByTestId("load-dataset-btn").click();
+      cy.getByTestId("dataset-fields-table");
+      cy.url().should("contain", "/dataset/demo_users_dataset");
+    });
+  });
+
+  describe("Dataset fields view", () => {
+    it("Can navigate to edit a dataset via URL", () => {
+      cy.visit("/dataset/demo_users_dataset");
+      cy.getByTestId("dataset-fields-table");
+    });
+
+    it("Can choose a different collection", () => {
+      cy.visit("/dataset/demo_users_dataset");
+      cy.getByTestId("field-row-price").should("not.exist");
+      cy.getByTestId("collection-select").select("products");
+      cy.getByTestId("field-row-price").should("exist");
+    });
+
+    it("Can render an edit form for a dataset field with existing values", () => {
+      cy.visit("/dataset/demo_users_dataset");
+      cy.getByTestId("field-row-uuid").click();
+      cy.getByTestId("edit-drawer-content");
+      cy.getByTestId("description-input").should(
+        "have.value",
+        "User's unique ID"
+      );
+      cy.getByTestId("identifiability-input").should("contain", "Identified");
+      cy.getByTestId("selected-categories").children().should("have.length", 1);
+      // cypress has trouble with testid's that have special characters, so put it in quotes and use regular cy.get
```

Comment:
This is caused by the [custom command](https://github.com/ethyca/fides/blob/ab27caf89d38e409de92507e423540d17a2b9f2a/clients/admin-ui/cypress/support/commands.ts#L4) right? So would wrapping the interpolated string in quotes solve this in general?

Reply:
yay nice!